cmd/kube-proxy/app/server.go

```diff
@@ -380,12 +397,34 @@ func (o *Options) loadConfigFromFile(file string) (*kubeproxyconfig.KubeProxyCon
 	return o.loadConfig(data)
 }
 
-// loadConfig decodes data as a KubeProxyConfiguration object.
+// loadConfig decodes a serialized KubeProxyConfiguration to the internal type.
 func (o *Options) loadConfig(data []byte) (*kubeproxyconfig.KubeProxyConfiguration, error) {
+
+	var (
+		configObj runtime.Object
+		gvk       *schema.GroupVersionKind
+	)
+
```

Comment:
I don't think, that you need this block here.

Reply:
Ok, cleaned up.